### PR TITLE
Exploit command injection

### PR DESCRIPTION
**Pull request by IlyasShabi**
## Motivation

Add test for command injection exploit in Node.js

## Changes

- Add Node.js endpoints
- Support Test_Shi_Telemetry_Variant_Tag test
- Add documentation

## Workflow

1. ⚠️ Create your PR as draft ⚠️
2. Work on you PR until the CI passes (if something not related to your task is failing, you can ignore it)
3. Mark it as ready for review
    * Test logic is modified? -> Get a review from RFC owner. We're working on refining the `codeowners` file quickly.
    * Framework is modified, or non obvious usage of it -> get a review from [R&P team](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X)

:rocket: Once your PR is reviewed, you can merge it!

🛟 [#apm-shared-testing](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X) 🛟

## Reviewer checklist

* [ ] If PR title starts with `[<language>]`, double-check that only `<language>` is impacted by the change
* [ ] No system-tests internal is modified. Otherwise, I have the approval from [R&P team](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X)
* [ ] CI is green, or failing jobs are not related to this change (and you are 100% sure about this statement)
* [ ] A docker base image is modified?
    * [ ] the relevant `build-XXX-image` label is present
* [ ] A scenario is added (or removed)?
    * [ ] Get a review from [R&P team](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X)

### CODE REVIEW

**Review by cbeauchesne — CHANGES_REQUESTED** [review]
CI is failing.

Also, small pitfall of system-tests : CI is terribly long. So there are mechanism to shorten it if needed. One of them is by prefixing your PR title by a language name -> it will run only this language tests.

But if your PR changes a test, you may break other languages CI if you don't check it 😱 

So the rules of thumb is : a test is modified => no prefix title.

I've updated it 😃

File: tests/appsec/rasp/test_cmdi.py
Comment (jandro996) [inline tests/appsec/rasp/test_cmdi.py:1]: Hey @IlyasShabi have a look to my PR(https://github.com/DataDog/system-tests/pull/3691) related with this test, we are pretty aligned, only one thing it's different and is as the cmdi waf heuristics are done to support arrays it's better to use arrays intead of Strings in the requests if is possible

File: docs/weblog/README.md [hunk 1]
```diff
@@ -759,6 +759,28 @@ Examples:
 - `GET`: `/rasp/shi?list_dir=$(cat /etc/passwd 1>&2 ; echo .)
 - `POST`: `{"list_dir": "$(cat /etc/passwd 1>&2 ; echo .)"}`
 
+### \[GET,POST\] /rasp/cmdi
+
+This endpoint is used to test for command injection attacks by executing a command without launching a shell.
+The chosen operation must be injected with the `GET` or `POST` parameter.
+
+Query parameters and body fields required in the `GET` and `POST` method:
+- `command`: containing the string to be executed as a command.
```
Comment (jandro996) [inline docs/weblog/README.md:768]: string array if its possible

**Review by jandro996 — CHANGES_REQUESTED** [review]
LGTM! but check my comment related with the arrays in the test_cdmi.py file

File: tests/appsec/rasp/test_cmdi.py
Comment (IlyasShabi) [reply tests/appsec/rasp/test_cmdi.py:1]: @jandro996 yes I agree with you. I will update my PR

File: tests/appsec/rasp/test_cmdi.py [hunk 2]
```diff
@@ -30,8 +30,14 @@ def test_cmdi_get(self):
             self.r,
             "rasp-932-110",
             {
-                "resource": {"address": "server.sys.exec.cmd", "value": "/bin/evilCommand"},
-                "params": {"address": "server.request.query", "value": "/bin/evilCommand"},
+                "resource": {
+                    "address": "server.sys.exec.cmd",
+                    "value": "/usr/bin/reboot -f",
```
Comment (NachoEchevarria) [inline tests/appsec/rasp/test_cmdi.py:35]: If a test fails blocking the command call, would the host machine reboot?

Comment (NachoEchevarria) [reply tests/appsec/rasp/test_cmdi.py:35]: Yes, while as @jandro996 suggested, the reboot command will not usually run without admin permissions, using /usr/bin/touch seems like a good and safer choice. Thanks!

Comment (IlyasShabi) [reply tests/appsec/rasp/test_cmdi.py]: Done

**Review by jandro996 — APPROVED** [review]
LTGM!

**Review by uurien — APPROVED** [review]

**Review by cbeauchesne — APPROVED** [review]

**Pull request merged**